code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -54,7 +54,7 @@
 	new /obj/item/stock_parts/cell/potato(Tsec)
 	var/obj/item/reagent_containers/food/drinks/drinkingglass/shotglass/S = new(Tsec)
 	S.reagents.add_reagent("whiskey", 15)
-	S.on_reagent_change()
+	S.on_reagent_change(ADD_REAGENT)

```

Comment:
wait this is stupid, add_reagent calls this

Reply:
🤔

Reply:
Just remove it then.

Reply:
nah can't be bothered to confirm it works after still